operatorinterface.py

```diff
@@ -85,11 +102,11 @@ def getAxisBindingOfName(name: str) -> AnalogInput:
             constants.kDriveToTargetControlButtonName
         )
 
-        self.toggleIntakeControl = getButtonBindingOfName(
-            constants.kToggleIntakeButtonName
+        self.deployIntakeControl = getAxisBindingOfName(
```

Comment:
Where is getAxisBindingOfName? I can't find it in the imports or anything

Reply:
https://github.com/1757WestwoodRobotics/2022-RapidReact/blob/6b155eeb0e49a3e50b4507bccf5f6c468382a0dc/operatorinterface.py#L71